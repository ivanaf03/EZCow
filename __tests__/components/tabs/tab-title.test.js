import React from "react";

import { render } from "@testing-library/react-native";

import TabTitle from "../../../components/tabs/tab-title";

describe("TabTitle", () => {
  it("renders correctly", () => {
    const tree = render(<TabTitle text="Test" />);
    expect(tree).toMatchSnapshot();
  });
});