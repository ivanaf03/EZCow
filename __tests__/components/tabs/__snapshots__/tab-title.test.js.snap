// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabTitle renders correctly 1`] = `
<Text
  style={
    [
      {
        "marginTop": 16,
      },
      {
        "marginLeft": 16,
        "marginRight": 16,
      },
      {
        "color": "#ffffff",
      },
      {
        "fontSize": 36,
        "lineHeight": 40,
      },
      {
        "fontFamily": "Nunito-Bold",
      },
      {
        "borderBottomWidth": 2,
      },
      {
        "borderBottomColor": "#ffffff",
        "borderLeftColor": "#ffffff",
        "borderRightColor": "#ffffff",
        "borderTopColor": "#ffffff",
      },
    ]
  }
>
  Test
</Text>
`;
